hq/app/aws/AWS.scala

```diff
@@ -1,13 +1,26 @@
 package aws
 
+import com.amazonaws.ClientConfiguration
 import com.amazonaws.auth.profile.ProfileCredentialsProvider
 import com.amazonaws.auth.{AWSCredentialsProviderChain, STSAssumeRoleSessionCredentialsProvider}
+import com.amazonaws.client.builder.AwsClientBuilder
+import com.amazonaws.regions.Regions
+import com.amazonaws.services.cloudformation.{AmazonCloudFormationAsync, AmazonCloudFormationAsyncClientBuilder}
+import com.amazonaws.services.ec2.{AmazonEC2Async, AmazonEC2AsyncClientBuilder}
+import com.amazonaws.services.identitymanagement.{AmazonIdentityManagementAsync, AmazonIdentityManagementAsyncClientBuilder}
+import com.amazonaws.services.inspector.{AmazonInspectorAsync, AmazonInspectorAsyncClientBuilder}
+import com.amazonaws.services.support.{AWSSupportAsync, AWSSupportAsyncClientBuilder}
+import config.Config
 import model.AwsAccount
+import play.api.Configuration
 import utils.attempt.{Attempt, Failure}
 
 
 object AWS {
-  def credentialsProvider(account: AwsAccount): AWSCredentialsProviderChain = {
+
+  //  availableRegions <- EC2.getAvailableRegions(regionClient)
```

Comment:
What's this for?

Reply:
Damn, that was a reminder to look at available regions.  I'll remove.